software/rosariosis.yml

```
@@ -1,6 +1,6 @@
 name: RosarioSIS
 website_url: https://www.rosariosis.org/
-description: RosarioSIS, free Student Information System for school management.
+description: Free Student Information System for school management.
```

```suggestion
description: Student Information System for school management. Features students demographics, grades, scheduling, attendance, student billing, discipline & food service modules.
```

Free can be removed for sure, as this is implied when listed here.
Maybe also some more information what a SIS is?